bindings/GnosisSafe/GnosisSafe.go

```diff
@@ -0,0 +1,6777 @@
+// This file was generated by seer: https://github.com/moonstream-to/seer.
+// seer version: 0.1.20
+// seer command: seer evm generate --package GnosisSafe --cli --struct GnosisSafe --output bindings/GnosisSafe/GnosisSafe.go
+// Code generated - DO NOT EDIT.
+// This file is a generated binding and any manual changes will be lost.
+
+package GnosisSafe
```

Comment:
`safe-gobal/safe-smart-accounts` repository has two Safe contracts defined at the top level:
- `Safe`
- `SafeL2`

[link](https://github.com/safe-global/safe-smart-account/tree/3e2ee891c0adb65cec4e2f81782d9f646130c715/contracts)

What does this contract correspond to? Can it be used on L1 and L2s (and L3s)?

Reply:
This is corresponding to [GnosisSafe-V1.2.0.sol](https://github.com/safe-global/safe-core-sdk/blob/main/packages/protocol-kit/contracts/safe_V1_2_0/GnosisSafe.sol)

Reply:
For now, we are just using it to retrieve the Safe Nonce

Reply:
Sounds good to me.